code/datums/quirks/negative.dm

```diff
@@ -98,7 +98,7 @@
 	 */
 /datum/quirk/item_quirk/brainproblems
 	name = "Brain Tumor"
-	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some mannitol!"
+	desc = "You have a little friend in your brain that is slowly destroying it. Better bring some Mannitol!"
```

Comment:
again, no capitalization needed

Reply:
but it's referring to a proper noun, Mannitol

Reply:
Medicines are not proper nouns, they’re just trivial names to easily refer to compounds. It’s not a person, place or thing